masonry/src/tracing_backend.rs

```diff
@@ -34,11 +35,15 @@ pub(crate) fn try_init_wasm_tracing() -> Result<(), SetGlobalDefaultError> {
 }
 
 #[cfg(not(target_arch = "wasm32"))]
-pub(crate) fn try_init_layered_tracing() -> Result<(), SetGlobalDefaultError> {
-    // Default level is DEBUG in --dev, INFO in --release
+pub(crate) fn try_init_layered_tracing(
+    default_level: Option<LevelFilter>,
+) -> Result<(), SetGlobalDefaultError> {
```

Comment:
```suggestion
fn try_init_layered_tracing(
    default_level: LevelFilter,
) -> Result<(), SetGlobalDefaultError> {
```
I'd like us to move the fallback cases into `try_init_tracing` instead. This would also allow the WASM version to have the same signature and avoid duplicating the fallback logic.

Reply:
> I'd slightly tweak the separation of concerns, but it seems like you want this to be a different way

The advantage of doing things this way is that it keeps all the comments in a single function.

(Also, meta comment. For me seeing a suggestion block communicates "This is a self-contained change you can commit right now." I'd advise not using them for more open-ended suggestions that will fail to compile without other changes. Regular code blocks are fine for that.)